simulacrum/types.py

```diff
@@ -0,0 +1,125 @@
+"""Functions for generating random data."""
+
+from uuid import uuid4
+import logging
+import sys
+import datetime
+import numpy as np
+import pandas as pd
+from faker import Faker
+
+FAKE = Faker()
+DT_CLASSES = (datetime.datetime, datetime.date)
+
```

Comment:
That's pretty. It's typically my code style. 
I prefer to see these functions than the original contributor's classes.

Well done